configs/components/rubygem-deep-merge.rb

```diff
@@ -7,9 +7,15 @@
 
   pkg.build_requires "ruby"
 
+  pkg.environment "PATH" => "#{settings[:bindir]}:$$PATH"
+
   # Because we are cross-compiling on sparc, we can't use the rubygems we just built.
   # Instead we use the host gem installation and override GEM_HOME. Yay?
-  pkg.environment "GEM_HOME" => settings[:gem_home]
+  if platform.is_windows?
+    pkg.environment "GEM_HOME" => platform.convert_to_windows_path(settings[:gem_home])
```

Comment:
why not set GEM_HOME for windows in the project config and do the conditional once instead of doing it for all the gems?